### PR TITLE
201223

### PR DESCRIPTION
**Pull request by George-ast**

### CODE REVIEW

File: home5.1.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+people_records.append(new_record)
+
+substitution = people_records[1]
+people_records[1] = people_records[5]
+people_records[5] = substitution
+
+print(substitution)
+print(people_records)
+
+if (people_records[6][2] > 30
```
Comment (antohaUa) [inline home5.1.py]: ./T_HOMETSAKS/Lesson_05/05_GeorgiyAstashenok.py:43:9: W503 line break before binary operator
./T_HOMETSAKS/Lesson_05/05_GeorgiyAstashenok.py:44:9: W503 line break before binary operator

File: home5.1.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    ('Daniel', 'Taylor', 38, 'Architect', 'Portland'),
+    ('Grace', 'Moore', 25, 'Graphic Designer', 'Miami'),
+    ('Samuel', 'Jones', 50, 'Business Consultant', 'Atlanta'),
+    ('Emma', 'Hall', 31, 'Chef', 'Dallas'),
+    ('William', 'Clark', 29, 'Financial Analyst', 'Houston'),
+    ('Ava', 'White', 42, 'Journalist', 'San Diego'),
+    ('Ethan', 'Anderson', 36, 'Product Manager', 'Phoenix'),
+]
+new_record = ('Olga', 'Black', 43, 'Student', 'Odesa')
+people_records.append(new_record)
```
Comment (antohaUa) [inline home5.1.py]: This operation will add record to the end. But you need to at at the beginning

File: home5.1.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+substitution = people_records[1]
+people_records[1] = people_records[5]
+people_records[5] = substitution
```
Comment (antohaUa) [inline home5.1.py]: Python allows to do this in one line

File: home5.1.py (hunk 1 above)
Comment (antohaUa) [inline home5.1.py]: Do not forget about hardcodes and `all()` ,` any()` constructions

**Review by antohaUa — CHANGES_REQUESTED** [review]
Fixes required

File: home5.1.py (hunk 1 above)
Comment (George-ast) [reply home5.1.py]: W503 line break before binary operator  у меня нет такой ошибки.  Прошелся еще раз flake8

File: home5.1.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    ('Grace', 'Moore', 25, 'Graphic Designer', 'Miami'),
+    ('Samuel', 'Jones', 50, 'Business Consultant', 'Atlanta'),
+    ('Emma', 'Hall', 31, 'Chef', 'Dallas'),
+    ('William', 'Clark', 29, 'Financial Analyst', 'Houston'),
+    ('Ava', 'White', 42, 'Journalist', 'San Diego'),
+    ('Ethan', 'Anderson', 36, 'Product Manager', 'Phoenix'),
+]
+new_record = ('Olga', 'Black', 43, 'Student', 'Odesa')
+people_records.insert(0, new_record)
+"""Modified list."""
```
Comment (antohaUa) [inline home5.1.py:34]: for comments better to use lines started with '#'

File: home5.1.py [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+people_records[1], people_records[5] = people_records[5], people_records[1]
+
+print(people_records)
+
+indexes_check = [6, 10, 13]
+result = all(people_records[i][2] >= 30 for i in indexes_check)
```
Comment (antohaUa) [inline home5.1.py:40]: 30 is hardcode - better to move it to the top as constant

File: home5.1.py [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+    ('Ethan', 'Anderson', 36, 'Product Manager', 'Phoenix'),
+]
+new_record = ('Olga', 'Black', 43, 'Student', 'Odesa')
+people_records.insert(0, new_record)
+"""Modified list."""
+people_records[1], people_records[5] = people_records[5], people_records[1]
+
+print(people_records)
+
+indexes_check = [6, 10, 13]
```
Comment (antohaUa) [inline home5.1.py:39]: These indexes also hardcodes - aka initial data - better to move to the top of the module as constant

File: home5.1.py [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+result = all(people_records[i][2] >= 30 for i in indexes_check)
+
+if result:
+    print(True)
+else:
+    print(False)
```
Comment (antohaUa) [inline home5.1.py:45]: NO sense cause result already has boolean value
just `print(result)`

**Review by antohaUa — APPROVED** [review]
Good

**Pull request merged**